index.html

```diff
@@ -3,14 +3,45 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title>Rock, Paper, Scissors</title>
   </head>
 
   <body>
 
+    <div class="container">
+
+      <div id="myBtn" class="player_controls">
+        <button id="rock" class="button_rock"></button>
+        <button id="paper" class="button_paper"></button>
+        <button id="scissors" class="button_scissors"></button>
+      </div>
+
+      <div class="user_messages">
+        <p id="beginMsg">Click a choice to begin rock, paper, scissors</p>
+        <p id="roundResult"></p>
+        <p id="finalScore"></p>
+      </div>
+      
+      
+      <button id="resetGame" class="reset_button"></button>
+    
+    </div>
+
     <script>
+
+      let playerScore = parseInt(0);
+      let computerScore = parseInt(0);
+      const buttons = document.getElementById('myBtn');
```

Comment:
Once you change the ID's to use `data-behavior` you will need to use `.querySelector()` instead https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector